src/modules/translation/languages/en.json

```diff
@@ -611,16 +611,16 @@
   "land_assign_ens_page": {
     "title": "Assign {name}",
     "subtitle": "Assign a name to {land}",
-    "set_name_message": "In order to set this name to your parcel you first need to send 2 transactions.",
+    "set_name_message": "Before you can assign this name to your parcel, you need to submit 2 transactions: one that translates the human readable, alphanumeric name into an Ethereum address and one that creates a hash code linking that generated address to your LAND.",
     "set_resolver": "Set Resolver",
-    "set_resolver_explanation": "The Resolver is a Smart Contract responsible for the process of translating names into addresses.",
+    "set_resolver_explanation": "This translates your name into an Ethereum address.",
     "set_content": "Set Content",
-    "set_content_explanation": "The Content is a hash refering a redirection to your land page that will be saved on your Resolver."
+    "set_content_explanation": "This creates a redirect to your LAND page and stores it in the Resolver."
   },
   "ens_select_land_page": {
     "title": "Assign {name}",
-    "subtitle": "You can assign this name to any of your Lands",
-    "land_selected": "Land selected:",
+    "subtitle": "You can assign this name to any of your parcels or estates.",
```

Comment:
The name can only be assigned to parcels @nicosantangelo @malaniz

Reply:
mm you are not seeing Estates because the legacy tiles api is not working in `.zone` and is necessary for generating the estates in memory, but if you switch to a working api you should be able to assign to estates to, no?

Reply:
Hmmm, It depends because you can't go to the explorer with an estate Id:
`play.decentraland.org?{estate_id}`

You need to link your name to a parcel inside the estate to have something like

`play.decentraland.org?{x},{y}`

Reply:
I've changed all the text to refer only Parcels.